test/util/utils.js

```diff
@@ -138,7 +138,20 @@ exports.getPullRequestNumber = function getPullRequestNumber() {
  */
 exports.getRepoName = function getRepoName() {
   let result = process.env['TRAVIS_REPO_SLUG'];
-  console.log(`@@@@@ process.env['TRAVIS_REPO_SLUG'] - ${process.env['TRAVIS_REPO_SLUG']}`);
+  console.log(`@@@@@ process.env['TRAVIS_REPO_SLUG'] - ${result}`);
+
+  return result;
+};
+
+/**
+ * Gets the source repo name for PR's. We are using the environment
+ * variable provided by travis-ci. It is called TRAVIS_PULL_REQUEST_SLUG. More info can be found here:
+ * https://docs.travis-ci.com/user/environment-variables/#Convenience-Variables
+ * @returns {string} repo name or 'undefined'.
+ */
+exports.getSourceRepoName = function getSourceRepoName() {
```

Comment:
minor: you may want to call the method getPrSourceRepoName, just to be more clear which source you're referring to.